docs/general/main.md

```diff
@@ -0,0 +1,33 @@
+# Overview
```

Comment:
This seems like a really long section of text mostly describing the purpose of documentation, most of which I don't think we need. I'd keep an adjusted version of the first sentence and maybe add some examples of the kinds of operations we're talking about (or just point to the API docs).

I'd also just move whatever remains to the readme and get rid of this file/section.

Reply:
I think for now I will move the majority of this as a comment to the `README` until we have time to fully flesh it out in a separate PR.